src/main/java/seedu/address/model/module/Module.java

```diff
@@ -87,10 +97,9 @@ public int hashCode() {
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
-        builder.append(getIdentifier())
-                .append(" has ")
-                .append(groups.size())
-                .append(" groups");
+        builder.append(getName())
+                .append(" with Module Code ")
+                .append(getIdentifier());
```

Comment:
Remember to update this later to a printed array format (i.e "[ apple, banana, carrot]").